ema/nextjs/src/components/dotcms-page.js

```diff
@@ -56,16 +58,29 @@ export const DotcmsPage = () => {
             }
         },
         [router]
-    ); // We need this because otherwise the function will create on every render otherwise and we cannot create the function outside because it's bounded to the router and to the ref
+    );
     // We need to unbound this from the component, with a custom hook maybe?
 
+    const eventScrollHandler = useCallback(
+        (event) => {
+            window.parent.postMessage(
+                {
+                    action: 'scroll'
+                },
+                '*'
+            );
+        },
+        [router]
```

Comment:
We don't need the router in the dependencies array

Reply:
You added that :D

Reply:
Oh